Makefile

```diff
@@ -1,6 +1,9 @@
 SOURCEDIR=.
 SOURCES = $(shell find $(SOURCEDIR) -name '*.go')
-VERSION=$(shell git describe --always --tags)
+VERSIONDATE:= $(shell date -Is)
+VERSION:=$(shell git describe --always --tags)
+VERSION:=$(addsuffix -MODIFIED, $(VERSION))
+#VERSION:=$(addsuffix $(VERSIONDATE), $(VERSION))
```

Comment:
@ranjib This change is required?

Reply:
@ranjib @prsws What are the purpose of this binary files?

![Screenshot from 2021-01-31 11-33-10](https://user-images.githubusercontent.com/4323228/106381336-481e2700-63b8-11eb-86e7-52c7179d456a.png)

Reply:
Hector:

This is my first time contributing to an open source project, so please excuse my newbie mistakes. I use the Makefile VERSION statements to identify my version as different from upstream when I post it to other forums such as Robo-Tank. In addition, the binary files are artifacts present in my development environment and were included in the merge unintentionally. Please let me know what else I need to clean up from my branch "uichanges2upstream" to make it proper for inclusion in the project. Thank you.